cdc/puller/sorter/backend_pool.go

```diff
@@ -71,6 +74,17 @@ func newBackEndPool(dir string, captureAddr string) *backEndPool {
 		filePrefix:        fmt.Sprintf("%s/sort-%d-", dir, os.Getpid()),
 	}
 
+	err := ret.lockSortDir()
+	if err != nil {
+		log.Error("failed to lock file prefix", zap.String("prefix", ret.filePrefix))
+		return nil
+	}
```

Comment:
The unified sorter would not work if lock prefix failed and backend pool is nil. Besides the error is not logged anywhere

Reply:
will fix later